src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
```

Comment:
we usually use rgba only when we need to change the opacity, so better to use here the HEX

Reply:
I'll take this into account, thanks for your help!